flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/decorators/KerberosMountDecoratorTest.java

```diff
@@ -19,39 +19,66 @@
 package org.apache.flink.kubernetes.kubeclient.decorators;
 
 import org.apache.flink.configuration.SecurityOptions;
+import org.apache.flink.kubernetes.KubernetesTestUtils;
 import org.apache.flink.kubernetes.kubeclient.FlinkPod;
 import org.apache.flink.kubernetes.kubeclient.KubernetesPodTestBase;
 import org.apache.flink.kubernetes.kubeclient.parameters.AbstractKubernetesParametersTest;
 import org.apache.flink.kubernetes.utils.Constants;
 
 import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.Volume;
 import io.fabric8.kubernetes.api.model.VolumeMount;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 import java.io.IOException;
+import java.util.List;
 import java.util.stream.Collectors;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
 
 /** General tests for the {@link KerberosMountDecoratorTest}. */
+@RunWith(Parameterized.class)
 public class KerberosMountDecoratorTest extends KubernetesPodTestBase {
 
     private KerberosMountDecorator kerberosMountDecorator;
 
     private AbstractKubernetesParametersTest.TestingKubernetesParameters
             testingKubernetesParameters;
 
+    private String keytabFile;
+    private String krb5ConfFile;
+
+    @Parameterized.Parameters()
+    public static Object[][] parameters() {
+        return new Object[][] {
+            {KEYTAB_FILE, KRB5_CONF_FILE}, {"mykeytab", "mykrb5.conf"},
+        };
+    }
+
+    public KerberosMountDecoratorTest(String keytabFile, String krb5ConfFile) {
+        this.keytabFile = keytabFile;
+        this.krb5ConfFile = krb5ConfFile;
+    }
```

Comment:
It's probably not necessary to duplicate all the test cases with different keytab/krb5conf names. Wouldn't it be good enough to only test for arbitrary custom names? I don't see how these tests can fail with default names while success with custom names.

Reply:
Thanks for your review, I will change it. Only test for arbitrary custom names.

Reply:
@xintongsong Would you like to have another look ?